### ISSUE TITLE
Anti-H2B antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6611]
**Preferred term label:** Anti-H2B antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Anti-histone antibody positivity (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Comment by pnrobinson** [comment]
added
